docs/modules/external-data-stores/pages/external-data-stores.adoc

```diff
@@ -31,15 +32,110 @@ NOTE: The data store that you choose must be a centralized system that is access
 
 NOTE: The `MapStore` interface extends the `MapLoader` interface. Therefore, all methods and configuration parameters of the `MapLoader` interface are also available on the `MapStore` interface.
 
+== Configuring Connections to External Data Stores (beta)
```

Comment:
Only this information should be on the page. The Information about MapStore and MapLoader needs to remain where it was.

Reply:
ah, sorry, fixed